cli/authenticating-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,27 @@
+[id="authenticating-by-using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Authenticating by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: roxctl-cli-authentication
+
+toc::[]
+
+[role="_abstract"]
+For authentication, you can use an API token, your administrator password, or the `roxctl login` command.
+
+The following are some guidelines for the effective use of API tokens:
```

Comment:
nit: should this be a info box?

Reply:
this is actually abstract info added in an assembly, I don't we need an info box for this one. 
@kcarmichael08 Any thoughts?

Reply:
The abstract is a description of the module that is needed for the Customer Portal so it should be kept.